packages/tooltips/src/elements/Tooltip.spec.tsx

```diff
@@ -9,8 +9,8 @@ import React from 'react';
 import userEvent from '@testing-library/user-event';
 import { render, act, renderRtl } from 'garden-test-utils';
 import { DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
-
-import { Tooltip, ITooltipProps } from './Tooltip';
+import { Tooltip } from './Tooltip';
+import { ITooltipProps } from '../utils/types';
```

Comment:
The argument against `/types` is that there's a pretty strong pattern of only having `/elements`, `/styles`, `/utils` at the src root. (After that, our evolutionary conventions get a bit chaotic 😅). Is `/types` a strong enough concept to pull out to the src root? And do we end up creating an unnecessary decision between `/types/{file}.ts`, `types.ts`, and `/types/index.ts`? I guess this is why I opted for `/utils/types` – because I didn't really want to think through the ramifications for every package.

Reply:
I have been thinking about this since we first discussed this. I'm actually in favor of `types/` since it felt like `types` shouldn't categorically fit under `utils/`. Another reason why: this would make  it easier for `utils/`, `elements/` and `styles/` to `import '../types;'` instead of adding `import '../utils/types` (for `elements` and `styles`).

Reply:
Ok, I like it. Thanks for the feedback.